### DISCUSSION TITLE
cypress runner stop immediately if runner's browser loses focus

### DISCUSSION THREAD

**Discussion by touensure** [cypress-io/cypress#28502] [Questions and Help]
Hi, I just upgraded my cypress from 13.4.0 to 13.6.1, I find my tests become unstable(it is more likely to fail conpared with former cypress version). Moreover, my tests often take more than one hour to get completed, I need to do other work while leaving  cypress running in background, however, cypress runner stop immediately if runner's browser loses focus(this will not happen if I use a former-version cypress).
could anyone help with this issue? thanks a lot.

**Comment by touensure** [comment]
more info: cypress version 13.5.x also works fine. So the problem only exists in the newest 3.6.x version

**Reply by chris-gotenna** [reply]
Same with 13.6.3

**Reply by alevenshteyn** [reply]
I think this might be related to multi-tab support, but I agree, this is impeding us because I virtually cannot use my machine while I am running a test. As soon as I click anything while the test is running, test loses focus and stops. There should be a setting to allow me to overwrite this behavior.

**Comment by chris-gotenna** [comment]
I ended up downgrading Cypress for now to work around it as it was
unserviceable for us.

​​​​
  |
***@***.***
